Gemfile

```diff
@@ -55,6 +55,7 @@ if defined?(JRUBY_VERSION)
 
   gem 'activerecord-jdbc-adapter', '~> 1.3.9'
   gem 'activerecord-jdbcmysql-adapter', '~> 1.3.9'
+  gem 'activerecord-jdbcpostgresql-adapter', '~> 1.3.9'
```

Comment:
We should also add `gem 'pg'` in the `else` block below, for Ruby MRI (especially useful during development, where restarts are much faster than with JRuby).